miasm2/ir/symbexec.py

```diff
@@ -442,6 +442,7 @@ def emulbloc(self, irb, step=False):
         """
         for assignblk in irb.irs:
             if step:
+                print 'Instr', assignblk.instr
```

Comment:
I don't agree with you.
It is *always* defined. But It can be *None* actually. See the init code:
```python

class AssignBlock(object):
   ...
    __slots__ = ["_assigns", "_instr"]

    def __init__(self, irs=None, instr=None):
        ...
        if irs is None:
            irs = []
        self._instr = instr
        self._assigns = {} # ExprAff.dst -> ExprAff.src

```

So it will print `Instr None` which is acceptable in my opinion.